### PR TITLE
Homework3: Vladimir Natsevsky

### PR DESCRIPTION
**Pull request by JunglistMNSQ**

### CODE REVIEW

File: src/Homework3/1_fizz_buzz.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+for num in range(1, 100):
```
Comment (belovmd) [inline src/Homework3/1_fizz_buzz.py]: it doesn't solve task in description

Comment (belovmd) [inline src/Homework3/1_fizz_buzz.py]: missed task description

File: src/Homework3/2_list_practice.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+"""
+
+list1 = [a + b for a in 'ab' for b in 'bcd']
+print(list1)
+
+list2 = list1[::2]
+print(list2)
+
+list3 = [str(el + 1) + 'a' for el in range(4)]
```
Comment (belovmd) [inline src/Homework3/2_list_practice.py]: '1234' ?

Comment (belovmd) [inline src/Homework3/2_list_practice.py]: it creates wrong list right now

File: src/Homework3/4_couple_elements.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+"""
+
+list1 = [1, 2, 1, 7, 1, 2, 1, 7, 1, 1, 3,
+         4, 3, 5, 3, 7, 3, 4, 4, 5, 4, 3,
+         ]
+for el in set(list1):
+    print('Для значения {el} количество пар в списке равно {result}'
+          .format(el=el,
+                  result=(list1.count(el) * (list1.count(el) - 1)) // 2))
```
Comment (belovmd) [inline src/Homework3/4_couple_elements.py]: it runs count twice. it can be counted one time

**Review by belovmd — COMMENTED** [review]
Please address these issues

File: src/Homework3/1_fizz_buzz.py (hunk 1 above)
Comment (JunglistMNSQ) [reply src/Homework3/1_fizz_buzz.py]: done

Comment (JunglistMNSQ) [reply src/Homework3/1_fizz_buzz.py]: done

File: src/Homework3/2_list_practice.py (hunk 2 above)
Comment (JunglistMNSQ) [reply src/Homework3/2_list_practice.py]: done

File: src/Homework3/4_couple_elements.py (hunk 3 above)
Comment (JunglistMNSQ) [reply src/Homework3/4_couple_elements.py]: done

**Review by belovmd — COMMENTED** [review]
9

**Pull request closed without merging**